package.json

```diff
@@ -20,16 +20,16 @@
     "eslint": ">=8.0.0"
   },
   "dependencies": {
-    "@babel/eslint-parser": "^7.22.5",
+    "@babel/eslint-parser": "^7.22.9",
     "@babel/eslint-plugin": "^7.22.5",
     "@typescript-eslint/eslint-plugin": "^5.59.9",
     "@typescript-eslint/parser": "^5.59.9",
-    "babel-preset-enact": "^0.1.3",
-    "eslint": ">=8.0.0",
+    "babel-preset-enact": "^0.1.4",
+    "eslint": "^8.45.0",
```

Comment:
@hong6316 , i see that this dependency version jumped back and forth a frew times in the past month. is Andrei's change correct in this case?